suggestions/templates/suggestions/problemsuggestion_form.html

```diff
@@ -224,6 +227,13 @@ <h3 class="alert-heading">
             render_solution();
         });
 
+        {% if pk %}
+          $("#delete").click(function(e) {
+            e.preventDefault();
+            window.location = '{% url 'suggest-delete' pk %}';
+          })
+        {% endif %}
+
```

Comment:
This is pretty bad, I think you should just use a link rather than a `<button>`.

You can make it look like a button using `btn-link` class, but the underlying element should be `<a>` rather than this hack.